funds-manager/funds-manager-api/src/types/quoters.rs

```diff
@@ -132,85 +135,85 @@ impl ExecutionQuote {
     }
 
     /// Get the buy amount as a decimal-corrected string
-    pub fn get_decimal_corrected_buy_amount(&self) -> Result<f64, String> {
+    pub fn get_decimal_corrected_buy_amount(&self, chain: Chain) -> Result<f64, String> {
```

Comment:
While this does work, I feel it would be cleaner to specify a chain on the ExecutionQuote struct and access using `self.chain`. You could set it in the chain-aware `get_execution_quote_handler` or derive it from the LI.FI quote which contains a chain ID as per the [docs](https://apidocs.li.fi/reference/get_v1-quote).

Reply:
perhaps i'll try the latter approach here. i didn't want to include a `chain` on the quote itself because then we have to update SDKs to expect that when serializing/deserializing a quote, and this field is only needed by the funds manager. i could `#[serde(skip)]` it but then we have to provide some default chain to use when deserializing, which is undefined

Reply:
yeah the latter approach would require `chain` to be added to the ExecutionQuote class. i believe that would be the extent of it